Contributing.md

```diff
@@ -137,7 +150,7 @@ When you're finished with your work and are ready to have it merged into the mai
 6. Add a title to your pull request, and describe all the changes you're making.
 7. Submit the pull request.
 
-One of the site administrators will process your pull request. Your pull request will surface on the OfficeDev/<repo name> site under Issues. When the pull request is accepted, the issue will be resolved.
+One of the site administrators will process your pull request. Your pull request will surface on the [OfficeDev/office-js-docs-reference](https://github.com/OfficeDev/office-js-docs-reference/pulls) site under **Pull requests**. When the pull request is accepted, the issue will be resolved.
 
```

Comment:
Could we replace the <repo name> placeholders throughout with office-js-docs-reference? Similarly for examples using "microsoft-graph-docs".